src/main/java/seedu/divelog/logic/commands/PlanningCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.divelog.logic.commands;
+
+import seedu.divelog.logic.CommandHistory;
+
+import seedu.divelog.logic.commands.exceptions.CommandException;
+import seedu.divelog.model.Model;
+
+//author @Cjunx
+/**
+ * Enters or exit the Planning Mode
+ */
+public class PlanningCommand extends Command {
+
+    public static final String COMMAND_WORD_PLAN = "Newplan";
```

Comment:
consider using lowercase for commands.

Reply:
Also ideally, individual commands should be split up. `PlanningCommand` should implement one and _only_ one command at a time.